tests/integration/test_charm_agent.py

```diff
@@ -82,7 +82,7 @@ async def agent(ops_test: OpsTest, series: str):
     agent_app_name = "jenkins-slave"
     # Agent currently does not support xenial
     agent: Application = await ops_test.model.deploy(
-        agent_app_name, series=series if series != "xenial" else "bionic", channel="edge"
+        agent_app_name, series=series, channel="edge"
```

Comment:
I know this wasn't mentioned in the story, let's also switch to using the stable channel